packages/mobile/src/screens/app-drawer-screen/AppDrawerScreen.tsx

```diff
@@ -45,35 +47,38 @@ const AppStack = memo(function AppStack(props: AppTabScreenProps) {
   )
 })
 
-export const AppDrawerScreen = () => {
-  const [gesturesDisabled, setGesturesDisabled] = useState(false)
+export const AppDrawerScreen = memo(
```

Comment:
what does this memo improve?

Reply:
I noticed this screen was rerendering each time user swiped open drawer, so this is a test to see if we get smoother behavior. from what i can tell this wont lead to any bugs, but may be an unnecessary memo which we can remove later on

Reply:
sg! good finds!!!